Donsol/Game.swift

```diff
@@ -60,10 +54,17 @@ class Game: ObservableObject {
     }
     
     func generateRoom () {
+        room = (0..<4).compactMap { _ in deck.drawCard() }
     }
 
     
-    func selectCard(card: Card) {
+    func selectCard(card: CardValue) {
+        // flip the card
+        room = room.map { c in
```

Comment:
By updating the state of all the cards in the room here, we will cause the view to rerender with the updated `isFlipped` value.